libs/npm/core/package.json

```diff
@@ -14,6 +14,8 @@
   "unpkg": "dist/index.min.js",
   "types": "dist/index.d.ts",
   "dependencies": {
+    "@fortawesome/free-solid-svg-icons": "6.2.0",
```

Comment:
I'd prefer not to pull in another dependency.  Use an icon from `react-icons` library which we already reference.  It has font-awesome icons.

Reply:
Updated, thanks